khmer/khmer_args.py

```diff
@@ -389,18 +393,22 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0, _sysmem=None):
```

Comment:
Sure; the tradeoff is that the workaround would add technical complexity
to our testing framework :)